core/src/commonMain/kotlin/com/ustadmobile/core/view/ContentEntryList2View.kt

```diff
@@ -0,0 +1,30 @@
+package com.ustadmobile.core.view
+
+import com.ustadmobile.lib.db.entities.ContentEntry
+import com.ustadmobile.lib.db.entities.ContentEntryWithParentChildJoinAndStatusAndMostRecentContainer
+
+interface ContentEntryList2View: UstadListView<ContentEntry, ContentEntryWithParentChildJoinAndStatusAndMostRecentContainer> {
+
+    enum class ContentEntryListViewMode(val viewMode: Int) {
```

Comment:
Why does this have it's own enum instead of using the ones provided by UstadView?